department-of-veterans-affairs/va.gov-cms
# VACMS-1031 poc dash events et

## Description
See #1031 

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/77966921-0d3bf600-72b2-11ea-8f6d-7dd109cef839.png)

## QA steps
 - As an admin, go to `pittsburgh-health-care/events` and review listings.
 - Play with filters, and make sure things sort and filter as expected
